stdlib/public/Concurrency/linker-support/magic-symbols-for-install-name.c

```diff
@@ -22,116 +22,55 @@
-  RPATH_INSTALL_NAME_DIRECTIVE( 6, 2)
-  RPATH_INSTALL_NAME_DIRECTIVE( 6, 3)
-  RPATH_INSTALL_NAME_DIRECTIVE( 7, 0)
-  RPATH_INSTALL_NAME_DIRECTIVE( 7, 1)
-  RPATH_INSTALL_NAME_DIRECTIVE( 7, 2)
-  RPATH_INSTALL_NAME_DIRECTIVE( 7, 3)
-  RPATH_INSTALL_NAME_DIRECTIVE( 7, 4)
-  RPATH_INSTALL_NAME_DIRECTIVE( 7, 5)
-  RPATH_INSTALL_NAME_DIRECTIVE( 7, 6)
-#elif TARGET_OS_IPHONE
-  RPATH_INSTALL_NAME_DIRECTIVE( 7, 0)
-  RPATH_INSTALL_NAME_DIRECTIVE( 7, 1)
-  RPATH_INSTALL_NAME_DIRECTIVE( 8, 0)
-  RPATH_INSTALL_NAME_DIRECTIVE( 8, 1)
-  RPATH_INSTALL_NAME_DIRECTIVE( 8, 2)
-  RPATH_INSTALL_NAME_DIRECTIVE( 8, 3)
-  RPATH_INSTALL_NAME_DIRECTIVE( 8, 4)
-  RPATH_INSTALL_NAME_DIRECTIVE( 9, 0)
-  RPATH_INSTALL_NAME_DIRECTIVE( 9, 1)
-  RPATH_INSTALL_NAME_DIRECTIVE( 9, 2)
-  RPATH_INSTALL_NAME_DIRECTIVE( 9, 3)
-  RPATH_INSTALL_NAME_DIRECTIVE(10, 0)
-  RPATH_INSTALL_NAME_DIRECTIVE(10, 1)
-  RPATH_INSTALL_NAME_DIRECTIVE(10, 2)
-  RPATH_INSTALL_NAME_DIRECTIVE(10, 3)
-  RPATH_INSTALL_NAME_DIRECTIVE(11, 0)
-  RPATH_INSTALL_NAME_DIRECTIVE(11, 1)
-  RPATH_INSTALL_NAME_DIRECTIVE(11, 2)
-  RPATH_INSTALL_NAME_DIRECTIVE(11, 3)
-  RPATH_INSTALL_NAME_DIRECTIVE(11, 4)
-  RPATH_INSTALL_NAME_DIRECTIVE(12, 0)
-  RPATH_INSTALL_NAME_DIRECTIVE(12, 1)
-  RPATH_INSTALL_NAME_DIRECTIVE(12, 2)
-  RPATH_INSTALL_NAME_DIRECTIVE(12, 3)
-  RPATH_INSTALL_NAME_DIRECTIVE(12, 4)
-  RPATH_INSTALL_NAME_DIRECTIVE(12, 5)
-  RPATH_INSTALL_NAME_DIRECTIVE(13, 0)
-  RPATH_INSTALL_NAME_DIRECTIVE(13, 1)
-  RPATH_INSTALL_NAME_DIRECTIVE(13, 2)
-  RPATH_INSTALL_NAME_DIRECTIVE(13, 3)
-  RPATH_INSTALL_NAME_DIRECTIVE(13, 4)
-  RPATH_INSTALL_NAME_DIRECTIVE(13, 5)
-  RPATH_INSTALL_NAME_DIRECTIVE(13, 6)
-  RPATH_INSTALL_NAME_DIRECTIVE(13, 7)
-  RPATH_INSTALL_NAME_DIRECTIVE(14, 0)
-  RPATH_INSTALL_NAME_DIRECTIVE(14, 1)
-  RPATH_INSTALL_NAME_DIRECTIVE(14, 2)
-  RPATH_INSTALL_NAME_DIRECTIVE(14, 3)
-  RPATH_INSTALL_NAME_DIRECTIVE(14, 4)
-  RPATH_INSTALL_NAME_DIRECTIVE(14, 5)
-  RPATH_INSTALL_NAME_DIRECTIVE(14, 6)
-  RPATH_INSTALL_NAME_DIRECTIVE(14, 7)
-  RPATH_INSTALL_NAME_DIRECTIVE(14, 8)
-#elif TARGET_OS_OSX
-  RPATH_INSTALL_NAME_DIRECTIVE(10,  9)
-  RPATH_INSTALL_NAME_DIRECTIVE(10, 10)
-  RPATH_INSTALL_NAME_DIRECTIVE(10, 11)
-  RPATH_INSTALL_NAME_DIRECTIVE(10, 12)
-  RPATH_INSTALL_NAME_DIRECTIVE(10, 13)
-  RPATH_INSTALL_NAME_DIRECTIVE(10, 15)
-  RPATH_INSTALL_NAME_DIRECTIVE(10, 14)
-  RPATH_INSTALL_NAME_DIRECTIVE(11,  0)
-  RPATH_INSTALL_NAME_DIRECTIVE(11,  1)
-  RPATH_INSTALL_NAME_DIRECTIVE(11,  2)
-  RPATH_INSTALL_NAME_DIRECTIVE(11,  3)
-  RPATH_INSTALL_NAME_DIRECTIVE(11,  4)
-  RPATH_INSTALL_NAME_DIRECTIVE(11,  5)
-  RPATH_INSTALL_NAME_DIRECTIVE(11,  6)
-
- // Link against @rpath/libswift_Concurrency.dylib for macCatalyst < 15.0.
-SWIFT_RUNTIME_EXPORT const char ld_previous_macCatalyst
-__asm("$ld$previous$@rpath/libswift_Concurrency.dylib$$6$1.0$15.0$");
-
-const char ld_previous_macCatalyst = 0;
+#if TARGET_OS_OSX || TARGET_OS_MACCATALYST
+RPATH_PREVIOUS_DIRECTIVE(PLATFORM_MACOS, 10.9, 12.0)
```

Comment:
I think these versions should reflect the the back-deployment versions for concurrency: macOS 10.15, iOS 13.0, watchOS 6.0, tvOS 13.0